packages/ecr-credential-provider/ecr-credential-provider-tmpfiles.conf

```diff
@@ -0,0 +1 @@
+C /var/lib/kubelet/plugins/ecr-credential-provider - - - -
```

Comment:
We might need to unconditionally remove the old file too, unless tmpfiles.d overwrites this. Also, would be good to check perms to make sure they're `0755`.